api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:17-alpine
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm install
+
+EXPOSE 4000
+# required for docker desktop port mapping
+
+CMD ["node", "app.js"]
```

Comment:
This will get executed as the image will start
Because we don't want to run this command at the build time of image!